current-development-phase.md

```diff
@@ -22,16 +22,22 @@ We provide sketches of the generation, storage, and use of arbitrary secrets bel
 1. The asset owner can _store_ a secret, either on the key server or locally.
     1. Storage on a key server MUST occur via a mutually authenticated channel that satisfies confidentiality and integrity. 
     1. Local storage on the user's device should be secure.
-1. The asset owner can _retrieve_ a secret from the key server. A retrieved secret can be:
-    1. Stored locally in secure storage on the user's device.
-    1. Copied to the system clipboard.
-1. The asset owner may _import_ a secret to the key server in an appropriate format. 
+1. The asset owner can _retrieve_ a secret from the key server. 
+    1. Retrieval from a key server MUST occur via a mutually authenticated channel that satisfies confidentiality and integrity. 
+    1. The key retrieved MUST be a key associated to the authenticated user.
+    1. A retrieved secret can be:
+        1. Stored locally in secure storage on the user's device.
+        1. Copied to the system clipboard.
+1. The asset owner may _import_ a secret to the system in an appropriate format. 
     1. The default expected format is as bytes of the form ``len || secret``, where `len` is 1 byte that represents the length of the secret `secret`.
     1. The implementation may define other acceptable import formats.
-1. The asset owner may _export_ the secret from the key server. The export format for the key SHOULD allow for easy transfer of the key material to another digital asset management system, i.e., secrets should be portable.
+    1. The user may choose to store the imported secret locally or on the key server.
```

Comment:
Do you want to add a comment here along the lines of, "if it's stored on the key server, export requires a mutually authenticated channel that satisfies confidentiality and integrity"?

Reply:
Or if you want to reduce verbiage throughout, you could probably say "authenticated session" a' la point 2.

Reply:
Yes, I should add that.